.yamllint.yaml

```diff
@@ -8,5 +8,7 @@ rules:
   float-values: enable
   line-length: disable
   octal-values: enable
+  indentation:
```

Comment:
This is required because of [this](https://github.com/yaml/pyyaml/issues/234) bug, where the `- ` in `- amd64` is treated as 2 spaces of indentation by `pyyaml` but not by `yamllint`.